style.css

```diff
@@ -0,0 +1,46 @@
+@import url("/components/component.css");
+
+html {
+  font-size: 62.5%;
+}
```

Comment:
This should be taken care of by the CSS reset in the component library stylesheet.